### PR TITLE
Fix frequent error for case OCP-12765

### PR DESCRIPTION
**Pull request by wzheng1**
https://mastern-jenkins-csb-openshift-qe.cloud.paas.psi.redhat.com/job/runner-v3/187405/console
@openshift/devexp-qe Please review, thanks!

### CODE REVIEW

File: features/cli/oc_import_image.feature [hunk 1]
```diff
@@ -108,13 +108,13 @@ Feature: oc import-image related feature
...
     Then the output should match:
-      | .*deployment-example.* |
+      | .*deployment-example.* | 
     When I run the :get client command with:
       | resource_name   | deployment-example |
       | resource        | dc                 |
       | o               | yaml               |
     Then the output should match:
-      |[Ll]astTriggeredImage.*deployment-example@sha256.*|
+      | .*openshift/deployment-example@sha256.* |
```
Comment (liangxia) [inline features/cli/oc_import_image.feature:117]: `.*` means anything.  Perhaps we can just use `openshift/deployment-example@sha256` ?

File: features/cli/oc_import_image.feature [hunk 2]
```diff
@@ -108,13 +108,13 @@ Feature: oc import-image related feature
     When I run the :get client command with:
       | resource        | imagestreams |
     Then the output should match:
-      | .*deployment-example.* |
+      | .*deployment-example.* |
```
Comment (liangxia) [inline features/cli/oc_import_image.feature:111]: Looks like unexpected change ?

File: features/cli/oc_import_image.feature (hunk 1 above)
Comment (wzheng1) [reply features/cli/oc_import_image.feature]: Yeah, done!

File: features/cli/oc_import_image.feature (hunk 2 above)
Comment (wzheng1) [reply features/cli/oc_import_image.feature:111]: Change it by accident, but no idea why it cannot come back

Comment (liangxia) [reply features/cli/oc_import_image.feature:111]: Did you delete the white spaces at end of line and add it to the commit ?

File: features/cli/oc_import_image.feature [hunk 3]
```diff
@@ -147,7 +147,7 @@ Feature: oc import-image related feature
       | resource        | dc                 |
       | o               | yaml               |
     Then the output should match:
-      | [Ll]astTriggeredImage.*:.*<%= project.name %>\/deployment-example@sha256.*|
+      | <%= project.name %>\/deployment-example@sha256 |
```
Comment (liangxia) [inline features/cli/oc_import_image.feature:150]: The `/` is escaped there, but not in line 117.  Perhaps one of them will fail. Could you add test log ?

Comment (wzheng1) [reply features/cli/oc_import_image.feature:150]: No failure about it, the purpose is to test this difference.

Comment (wzheng1) [reply features/cli/oc_import_image.feature:150]: If they are the same, then it should be a bug.

File: features/cli/oc_import_image.feature (hunk 2 above)
Comment (wzheng1) [reply features/cli/oc_import_image.feature]: Fixed.

File: features/cli/oc_import_image.feature (hunk 3 above)
Comment (wzheng1) [reply features/cli/oc_import_image.feature:150]: Pass log is added to the description.

Comment (liangxia) [reply features/cli/oc_import_image.feature:150]: Ack

**Comment by liangxia** [comment]
/lgtm

Still need feature team to review the change though.

**Comment by liangxia** [comment]
/hold
to let feature team review.

**Pull request merged**